src/Microsoft.AspNetCore.Razor/Compilation/TagHelpers/TagHelperDescriptor.cs

```diff
@@ -200,5 +202,21 @@ public IEnumerable<TagHelperRequiredAttributeDescriptor> RequiredAttributes
         /// tag helper.
         /// </summary>
         public TagHelperDesignTimeDescriptor DesignTimeDescriptor { get; set; }
+
+        /// <summary>
+        /// A dictionary containing additional properties of the tag helper.
```

Comment:
`A dictionary containing additional information about the <see cref="TagHelperDescriptor"/>.`